anonymisations.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"strings"
+)
+
+type anonymisation func(string) string
+
+func identity(s string) string {
```

Comment:
I think all of these methods should have doc comments documenting their purpose as this basically serves as the list of things you can use.

Reply:
If you want to use comments as a form of documentation I'd recommend doing them in [godoc](https://blog.golang.org/godoc-documenting-go-code) style.

This will mean that if we ever make this project importable it'll be ready to go with a godoc and the change is simply to enter the function name at the start of each function/type description.